src/services/API/Video.js

```diff
@@ -0,0 +1,20 @@
+import { apiClient } from "@/services/API/RootClient.js";
+
+export default {
+  async createVideo(videoDetails) {
+    // API to create a video object in the DB
+    var response = await apiClient().post(
+      process.env.VUE_APP_BACKEND_VIDEOS,
+      videoDetails
+    );
+    return response.data.id;
+  },
+
+  updateVideo(videoDBId, videoDetails) {
+    // API to update a video object in the DB
+    return apiClient().patch(
+      process.env.VUE_APP_BACKEND_VIDEOS + videoDBId + "/",
```

Comment:
is this still needed or have we configured the trailing slashes to false?

Reply:
Tried the flag APPEND_SLASH=False in settings.py but was not able to get the desired behaviour. Endpoints were giving Page not found error, so didn't go ahead with that